content/en/docs/developer_guide/project/configuration/project_configuration.md

```diff
@@ -37,9 +37,10 @@ The following topics describe the project configuration settings.
 | disabled | Boolean | false | Set to `true` to display the generated Swagger API Docs. Changing the setting only works in production. Swagger documentation is always available in dev mode. |
 | prefix | String | '/apidoc' | Prefix for the API documentation. |
 | overrides | object | {} | Overrides to Swagger documentation. Any values set here do not change the functionality of the server, only what is exposed in the Swagger. |
-| overrides.host | string | \- | Hostname and optional port on which the server can be accessed. |
-| overrides.schemes | array | \- | Schemes which the server can be accessed using. Can be an array containing any of `http`, `https`, `ws`, or `wss`. |
+| overrides.host | string | \- | Hostname and optional port on which the server can be accessed. The value can be set to `null` to clear the `host`. |
+| overrides.schemes | array | \- | Schemes which the server can be accessed using. Can be an array containing any of `http`, `https`, `ws`, or `wss`. The value can be set to `null` to clear the `schemes`.|
 | overrides.basePath | string | \- | The root path on which the APIs hosted by the server are available. If provided, this must start with a leading slash (`/`). The value can be set to `null` to clear the `basePath`. |
+| overrides.servers | array | \- | OAS3 specific override. If provided, this must be an Array and each server should the vary least contain the server url as String. The value can be set to `null` to clear the `servers`. |
```

Comment:
Since this says it's OpenAPI 3 specific then the others should clarify they're OpenAPI 2 specific.

Reply:
Spelling mistake with "vary"
```suggestion
| overrides.servers | array | \- | OpenAPI 3 specific override. If provided, this must be an array of server objects, and each server must contain the server `url` as string. The value can be set to `null` to clear the `servers`. |
```